spirv_glsl.hpp

```diff
@@ -64,6 +65,149 @@ typedef uint32_t AccessChainFlags;
 
 class CompilerGLSL : public Compiler
 {
+	struct KHR_shader_subgroupSupportHelper
+	{
+		// lower enum value = greater priority
+		enum Candidate
+		{
+			NV_gpu_shader_5,
+			NV_shader_thread_group,
+			NV_shader_thread_shuffle,
+			ARB_shader_ballot,
+			ARB_shader_group_vote,
+			AMD_gcn_shader,
+
+			CandidateCount
+		};
+		static std::string getExtensionName(Candidate c)
```

Comment:
@HansKristian-Work do you prefer a static constexpr array?

Reply:
`static const char *[]` seems appropriate here.

Reply:
@Crisspl ?

Reply:
done